### DISCUSSION TITLE
Does Freetz support using FRITZ!Box as a network repeater?

### DISCUSSION THREAD

**Discussion by knxste** [Freetz-NG/freetz-ng#848] [Allgemein]
Hi All,

I own a FRITZ!box 7270 V3 and a 7320 and I would like to use them as wifi repeaters to extend my home network. The original firmware does not support such feature (7270 supports WDS, but that's not what I need, as it requires another FRITZ as base station). Searching the AVM web, I've seen that some (older) models had such feature.
Before investing time in understanding and using Freetz, I would like to know if the feature I need is actually supported

thanks
ste

**Comment by aus-sua** [comment]
Short answer:
Sorry, no,
freetz/freetz-NG does not change the AVM features of the AVM-WiFi system.

Longer answer:
I think, these [statements](https://www.ip-phone-forum.de/threads/freetz-f%C3%BCr-7270-v2-zur-verwendung-als-universal-repeater-urm-client-mode.271755/) have not changed even today (2023).
https://www-ip--phone--forum-de.translate.goog/threads/freetz-f%C3%BCr-7270-v2-zur-verwendung-als-universal-repeater-urm-client-mode.271755/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en

If necessary OpenWRT would be a possibility for your FRITZ!Boxes, would have to be checked.

**Comment by fda77** [comment]
I'm using a 7320 and 1200 with openwrt as reapeater. FOS is to old and outdated. With openwrt you get a recent kernel with full iptables, vlans, unlimited ssids and wpa3. Not working with openwrt: 2. lan port, telephone (dect, if it has dect, cant remember)
